processors/games/Quake.js

```diff
@@ -1,8 +1,168 @@
+/* eslint-disable camelcase */
 /*
-* Quakecraft
+ * Quakecraft
  */
+
+const utility = require('../../util/utility');
```

Comment:
I think using object destructing here would be a good practice, i.e. 
```js
const { getRatio } = require('../../util/utility');
```
so `getRatio` can be referenced directly. I'll update this to my code as well.